### DISCUSSION TITLE
Backback PRO testing license and live usage

### DISCUSSION THREAD

**Discussion by emabusi2** [Laravel-Backpack/community-forum#922] [Q&A (Help)] [answered]
I want (intend) to buy backpack pro license for a single project but I have a query. I would like to install backpack on a test source code first for testing as it is my first time then actually use in my intended project, will I need 2 licenses for that? or how can I achieve this with this with one license?

**Comment by pxpm** [comment, marked as answer]
Hey @emabusi2 

It's one licence per project (assuming a single license scenario). But you can have `testing.project.com`, `pre-testing.project.com` and `project.com` with the same license. No issues. 

The demo of our packages is online at https://demo.backpackforlaravel.com

Cheers

**Comment by emabusi2** [comment]
How about on localhost? Do I need multipe licenses to test? @pxpm

Also, If no. Does it mean I can use my license on multiple localhost then when am ready to go live buy another license or how does it work

**Reply by pxpm** [reply]
Hey @emabusi2 

If you have 10 developers working on a project, all of them can use the same license locally. 

What I see people doing, and it's fine, a Company buys one license for them to use on their own project/website. 
Develop all client projects using that license so they only share the "company token" with the developers working on the project. 
Then the company buy a license for the client project and put it online in the production/staging '.envs' with the "client token". 

Developers can still work locally using the "company token" on any project, and the site is live with the "client token". 

This also benefits if you use contractors to do the job, you don't share the client license with the contractor, you just share your company license with them. You can then re-generate you company token after the job is done. 

Hope my examples were clear enough to explain how it works. Here https://backpackforlaravel.com/pricing at the bottom you have a FAQ that may answer other questions may have.

Cheers

**Reply by emabusi2** [reply]
Thank you so much. Off to purchase now.
